extras/rpms/glusterd2.spec

```diff
@@ -0,0 +1,81 @@
+%global _dwz_low_mem_die_limit 0
+
+%global provider github
+%global provider_tld com
+%global project gluster
+%global repo glusterd2
+%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path %{provider_prefix}
+
+Name: %{repo}
+Version: 4.0dev
+Release: 7
+Summary: The GlusterFS management daemon (preview)
+License: GPLv2 or LGPLv3+
+URL: https://%{provider_prefix}
+Source0: https://%{provider_prefix}/archive/%{version}-%{release}/%{name}-%{version}-%{release}.tar.gz
+Source1: glusterd.toml
+
+ExclusiveArch: x86_64
+
+BuildRequires: golang >= 1.8.0
+BuildRequires: glide >= 0.12.0
+BuildRequires: git
+BuildRequires: mercurial
+BuildRequires: systemd
+
+Requires: glusterfs-server >= 3.11.0
+Requires(post): systemd
+Requires(preun): systemd
+Requires(postun): systemd
+
+%description
+Preview release of the next generation GlusterFS management framework and daemon, coming with GlusterFS-4.0
+
+%prep
+%setup -q -n %{name}-%{commit}
+
+%build
+mkdir -p src/%(dirname %{import_path})
+ln -s ../../../ src/%{import_path}
+
+pushd src/%{import_path}
+# Install vendored packages
+# TODO: See if we can build with unbundled packages
+make vendor-install
```

Comment:
I think this should be as part of make dist. Good if we can avoid network calls during RPM/packaging create.

This is fine for current release, we can take care of this issue later.